### PR TITLE
497 automate website

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

**Comment by pchelle** [comment]
Is there actually a difference between this action and __pages build and deployment with artifacts-next__ action ?
If not, we can drop this PR

**Comment by Yuri05** [comment]
@pchelle The code for devel and release website is almost identical; also also identical for other repositories (more or less only installed prerequisites differ).
I have moved the code to a reusable OSP-global workflow - s. the PR https://github.com/Open-Systems-Pharmacology/Workflows/pull/18
Once this PR is merged, the dev-website yaml can be reduced to this:

```
name: Update dev website
on:
  push: 
    branches:
      - develop    

# The action needs permissions to update the branch
permissions:
  contents: write
          
jobs:
  update_dev_website:
    uses: Open-Systems-Pharmacology/Workflows/.github/workflows/CreateGitHubPagesForR.yml@main
    with:
      pkgdown_mode: 'devel'
      extra_r_packages: 'github::Open-Systems-Pharmacology/OSPSuite.RUtils'
```

Similar for the release yaml.

**Comment by pchelle** [comment]
> pkgdown can only use images

I double checked the figures were correctly added.
From what I read of pkgdown, this is likely a false warning: https://github.com/r-lib/pkgdown/issues/2194

File: .github/workflows/pkgdown_setup.R
Comment (Yuri05) [inline .github/workflows/pkgdown_setup.R:1]: the file is not required anymore; the code was integrated into the global workflow

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**